openmdao/core/tests/test_group.py

```diff
@@ -514,6 +514,191 @@ def compute(self, inputs, outputs):
                          "Group (<model>): src_indices has been defined in both "
                          "connect('indep.x', 'C1.x') and add_input('C1.x', ...).")
 
+    def test_om_slice_in_connect(self):
```

Comment:
I haven't seen any tests that use a slicer with a negative stop.  Could we add that for both serial and MPI?